src/radicalpy/estimations.py

```diff
@@ -460,6 +421,85 @@ def k_ST_mixing(Bhalf: float) -> float:
     return -C.g_e * (C.mu_B * 1e-3) * Bhalf / C.h
 
 
+def k_electron_transfer(R: float, deltaG: float = -1, lambd: float = 1) -> float:
+    """Electron transfer rate.
+
+    The default values return the maximum electron transfer rate (lambd = -deltaG).
```

Comment:
That `lambd = - deltaG` is a bit confusing? Do you mean to say they have the opposite sign? Then you should write something like `lambd * deltaG < 0` or `sgn(lambda) = - sgn(deltaG)`. And why is the `a` missing from lambda (at the end)?